commown_devices/models/common.py

```diff
@@ -89,9 +158,11 @@ def internal_picking(
     dest_location,
     origin,
     date=None,
-    do_transfer=False,
+    picking=None,
 ):
-    """Create picking with tracked and untracked products"""
+    """Create picking with tracked and untracked products, if a picking is passed as an
+    argument it wil try to update the existing picking with new move_lines"""
```

Comment:
"argument it wil try to update the existing picking with new move_lines" > "argument, try to update the existing picking with new move lines"